orb/executors/default.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+  version:
+    type: string
+    default: "v11.11-stretch"
+docker:
+  - image: quay.io/reactiveops/ci-images:<<parameters.version>>
```

Comment:
I need a container that contains pluto cli, @sudermanjr do you know what I should use here?

Reply:
Ok I have made some changes to use this image, one thing I am unsure of (new to circleci) is if the file or folder we want to check will be mounted so that the docker image we are using can check those file/files. 

Essentially I see the `quay.io/fairwinds/pluto:v5.1.0` has an `ENTRYPOINT ["pluto"]` so for example, I am passing the command `detect-files ./K8s/Descriptors` will the executor (the pluto docker image) be able to see the files at the provided path?